### PR TITLE
Avatar: Java examples

### PR DESCRIPTION
**Pull request by anezthes**
**Notes**:
- The `company-logo.png` example uses `StreamResource`. Couldn't get image to show using just its URL. Now it feels needlessly complicated though.
- `MenuBar` (Java) example is not working. Screenshot of the console output:

menu-bar-errors

### CODE REVIEW

**Comment by anezthes** [comment]
@DiegoCardoso recommended using:
```
import '@vaadin/flow-frontend/menubarConnector'; // hidden-full-source-line
```
in `avatar-menu-bar.ts` which reduced the number of errors to:

image

File: frontend/demo/component/avatar/avatar-menu-bar.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import '../../init'; // hidden-full-source-line
+import '@vaadin/flow-frontend/menubarConnector'; // hidden-full-source-line
```
Comment (DiegoCardoso) [inline frontend/demo/component/avatar/avatar-menu-bar.ts:2]: Adding `contextMenuConnector` seems fix the issue with the example:

```suggestion
import '@vaadin/flow-frontend/menubarConnector'; // hidden-full-source-line
import '@vaadin/flow-frontend/contextMenuConnector'; // hidden-full-source-line
```

File: src/main/java/com/vaadin/demo/component/avatar/AvatarGroupMaxItems.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import com.vaadin.demo.domain.Person;
+import com.vaadin.flow.component.avatar.AvatarGroup;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.router.Route;
+import java.util.List;
+
+@Route("avatar-group-max-items")
+public class AvatarGroupMaxItems extends Div {
+
+  private List<Person> people = DataService.getPeople(6);
```
Comment (DiegoCardoso) [inline src/main/java/com/vaadin/demo/component/avatar/AvatarGroupMaxItems.java:14]: I noticed that this is different from the TS example and it makes sense to have more items than the maximum visible items value. I would suggest to align and use this same quantity on the TS example.

**Review by DiegoCardoso — CHANGES_REQUESTED** [review]

**Comment by DiegoCardoso** [comment]
The "Working in progress" admonition can be removed.

File: src/main/java/com/vaadin/demo/component/avatar/AvatarGroupMaxItems.java (hunk 2 above)
Comment (anezthes) [reply src/main/java/com/vaadin/demo/component/avatar/AvatarGroupMaxItems.java:14]: Oversight. Fixed!

File: frontend/demo/component/avatar/avatar-menu-bar.ts (hunk 1 above)
Comment (anezthes) [reply frontend/demo/component/avatar/avatar-menu-bar.ts:2]: Fixed!

File: frontend/demo/component/avatar/avatar-menu-bar.ts [hunk 3]
```diff
@@ -22,8 +22,7 @@ export class Example extends LitElement {
   private menuBarItems: MenuBarItem[] = [];
 
   async firstUpdated() {
-    const { people } = await getPeople({ count: 1 });
-    const [person] = people;
+    const [person] = await getPeople(1);
```
Comment (DiegoCardoso) [inline frontend/demo/component/avatar/avatar-menu-bar.ts:25]: This is using the old `getPeople` API. You can see on the change the new one.

There seems to be that after merging, the changes got messed up and the older versions of the files got picked in favor of the new changes. You can also see below the leading comas being removed. And, in some other TS files, the imports are being reverted to the old format when we didn't have the `Frontend` alias.

**Review by DiegoCardoso — CHANGES_REQUESTED** [review]
The branch needs to be rebased properly to fix the merging issues.

File: frontend/demo/component/avatar/avatar-menu-bar.ts (hunk 3 above)
Comment (jouni) [reply frontend/demo/component/avatar/avatar-menu-bar.ts:25]: I’ll see what I can do – I’ll try rebasing and fixing the conflicts.

**Review by jouni — APPROVED** [review]

**Pull request merged**